prow/getting_started_deploy.md

```diff
@@ -354,14 +354,14 @@ Now run the following to update the configmap.
 
 ```sh
 $ kubectl create configmap config \
-  --from-file=config.yaml=path/to/config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+  --from-file=config.yaml=path/to/config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
```

Comment:
why not `--dry-run=server`, thats the better check?

Reply:
Thank you for the suggestion, changed to --dry-run=server instead.